cmk/base/plugins/agent_based/fortisandbox_disk.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+# author: Oguzhan Cicek, OpenSource Security GmbH - oguzhan(at)os-s.de
+
+# Example Input:
+#[...]
+#.1.3.6.1.4.1.12356.118.3.1.1.0 v2.52-build0340 (GA)
+#.1.3.6.1.4.1.12356.118.3.1.2.0 2
+#.1.3.6.1.4.1.12356.118.3.1.3.0 4
+#.1.3.6.1.4.1.12356.118.3.1.4.0 260459760
+#.1.3.6.1.4.1.12356.118.3.1.5.0 710698
+#.1.3.6.1.4.1.12356.118.3.1.6.0 3754417
+#.1.3.6.1.4.1.12356.118.3.1.7.0 2
+#.1.3.6.1.4.1.12356.118.3.1.8.0 32908349
+#.1.3.6.1.4.1.12356.118.3.2.1.0 5.2.50534
+#.1.3.6.1.4.1.12356.118.3.2.2.0 2.4.20034
+#.1.3.6.1.4.1.12356.118.3.2.3.0 3.2.279
+#.1.3.6.1.4.1.12356.118.3.2.4.0 4.478
+#.1.3.6.1.4.1.12356.118.3.2.5.0 14.613
+#[...]
+
+# Example GUI Output:
+# OK	FortiSandbox		Disk used: 5.28% used (694 GiB of 3.58 TiB)
+
+from .agent_based_api.v1.type_defs import (
+    DiscoveryResult,
+    CheckResult,
+    StringTable,
+)
+from .agent_based_api.v1 import (
+    register,
+    render,
+    Service,
+    equals,
+    check_levels,
+    SNMPTree,
+)
+from typing import Mapping, List, Tuple
+
+Section = List[str]
+
+
+def parse_fortisandbox_disk(string_table: List[StringTable]) -> Section:
+    return string_table[0][0]
+
+
+def discovery_fortisandbox_disk(section: Section) -> DiscoveryResult:
+    yield Service()
+
+
+def check_fortisandbox_disk(params: Mapping[str, Tuple[float, float]],
+                            section: Section) -> CheckResult:
+    disk_usage_upper = params.get("disk_usage")
+    diskusage = 1024 * 1024 * float(section[0])
+    diskcap = 1024 * 1024 * int(section[1])
+    diskperc = float(diskcap / diskusage)
+    display_diskusage = render.bytes(diskusage)
+    display_diskcap = render.bytes(diskcap)
+    yield from check_levels(
```

Comment:
Here, you should use our common routine for checking filesystem or disk usages: utils.df.df_check_filesystem_single